lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -193,6 +193,12 @@ def client_rb_content(chef_server_url, node_name)
         # about the workstation's chef zero server
         uri = URI.parse(chef_server_url)
         if uri.scheme == 'chefzero' && uri.host == 'localhost'
+          if convergence_options[:policy_name] || convergence_options[:policy_group]
+            raise "You are running Chef Zero and have policyfiles enabled " +
+              "however Chef Zero does not currently support the use " +
+              "of policyfile nativemode which is required for this option. " +
+              "To use this feature you must use chef-server 12.1 plus or hosted chef."
```

Comment:
Would it make sense to provide a link to get chef server as well, https://downloads.chef.io/chef-server/?